oggm/core/climate_prepro.py

```diff
@@ -0,0 +1,122 @@
+"""Climate data and mass-balance computations"""
+# Built ins
+import logging
+import os
+import datetime
+import warnings
+# External libs
+import numpy as np
+import netCDF4
+import pandas as pd
+import xarray as xr
+from scipy import stats
+import salem
+from scipy import optimize as optimization
+# Locals
+from oggm import cfg
+from oggm import entity_task
+from oggm.core.climate import process_gcm_data
+# Module logger
+log = logging.getLogger(__name__)
+
+
+@entity_task(log, writes=['gcm_data'])
+def prepro_cesm_data(gdir, filesuffix='', fpath_temp=None, fpath_precc=None,
+                     fpath_precl=None):
+    """Processes and writes the climate data for this glacier.
+
+    This function is made for interpolating the Community
+    Earth System Model Last Millennium Ensemble (CESM-LME) climate simulations,
+    from Otto-Bliesner et al. (2016), to the high-resolution CL2 climatologies
+    (provided with OGGM) and writes everything to a NetCDF file.
+
+    Parameters
+    ----------
+    filesuffix : str
+        append a suffix to the filename (useful for ensemble experiments).
+    fpath_temp : str
+        path to the temp file (default: cfg.PATHS['gcm_temp_file'])
```

Comment:
Here (and below) I guess we should change if for `cfg.PATHS['cesm_temp_file']` . What do you think?